frontend/integration-tests/protractor.conf.ts

```diff
@@ -86,6 +86,12 @@ export const config: Config = {
     return new Promise(resolve => htmlReporter.afterLaunch(resolve.bind(this, exitCode)));
   },
   suites: {
+    crd: [
+      'tests/login.scenario.ts',
+      'tests/base.scenario.ts',
+      'tests/crd.scenario.ts',
```

Comment:
We should be adding these new tests to either e2e or olmFull suite so that they run against pull requests.

Reply:
Yes, updated.

Reply:
ok